### ISSUE TITLE
Need a way to disable STDOUT / STDERR

### ISSUE THREAD

**Issue by mdklatt** [mdklatt/httpexec#4]
The request should accept a `stdout` and/or `stderr` key as a bool to enable/disable output from the respective stream of the target command. If the key defaults to `True` the existing behavior will be preserved, but this should probably be opt-in due to the consequences of inadvertently transmitting possibly sizable and/or binary data over HTTP.

**Comment by mdklatt** [comment]
Does it make sense to merge this with `binary`? That would be breaking change, but that's okay because this will be a major release.

``` json
{
  "stdin": {"value": "abcd", "encode": "base64"},
  "stderr": {"capture": false, "encode": None},
  "stderr": {"capture": false, "encode": "base84"}, 
}

```

**Comment by mdklatt** [comment]
Issue resolved by commit cfb251153b93c13667d8d3258050952911628878.
